src/ftest.jl

```diff
@@ -0,0 +1,130 @@
+type FTestTable
+    ntests::Int
+    SSR1::Array{Float64, 1} # sum of squared residuals, model 1
+    SSR2::Array{Float64, 1} # sum of squared residuals, model 2
+    df1::Array{Int, 1} # degrees of freedom, model 1 residuals
+    df2::Array{Int, 1} # degrees of freedom, model 2 residuals
+    MSR1::Array{Float64, 1} # mean of squared residuals, model 1
+    MSR2::Array{Float64, 1} # mean of squared residuals, model 2
+    fstat::Array{Float64, 1} # f statistic
+    pval::Array{Float64, 1} # p value
+end
+
+function FTestTable(SSR1::Array{Float64, 1}, SSR2::Array{Float64, 1}, df1::Array{Int, 1}, df2::Array{Int, 1}, MSR1::Array{Float64, 1}, MSR2::Array{Float64, 1}, fstat::Array{Float64, 1}, pval::Array{Float64, 1})
+#Note: Is checking args in internal helper functions the desired behavior?
+@argcheck length(SSR1) == length(SSR2) == length(df1) == length(df2) == length(MSR1) == length(MSR2) == length(fstat) == length(pval) "F test values  must all be the same length"
```

Comment:
Should be indented since it's inside of a function. Also we try to keep the line lengths down to 92 characters at most (which is admittedly an arbitrary cutoff, but it's what's used in Base).